kuksa-client/kuksa_client/__main__.py

```diff
@@ -149,10 +147,11 @@ def subscriptionIdCompleter(self, text, line, begidx, endidx):
     VSS_COMMANDS = "Kuksa Interaction Commands"
     INFO_COMMANDS = "Info Commands"
 
-    ap_getServerAddr = argparse.ArgumentParser()
     ap_connect = argparse.ArgumentParser()
     ap_connect.add_argument(
-        '-i', "--insecure", default=False, action="store_true", help='Connect in insecure mode')
+        'server', help=f"VSS server to connect to. Format: protocol://host[:port]. \
```

Comment:
Do we need to state somewhere that default port is 55555 regardless of protocol.
Or do we want it to be smarter and let default port vary depending on protocol to match default port for databroker and val-server

Reply:
I did not state it but choose sensible default ports now. AS the client tells you exactly what it is doing before it connects, I think no need to explicitly spell it out in usage message